torch/csrc/jit/passes/alias_analysis.cpp

```diff
@@ -366,6 +367,11 @@ void AliasDb::analyze(Node* node) {
       continue;
     }
 
+    // If this type cannot alias, continue. Can occur with a VarType schema
+    if (!shouldAnnotate(actualValue)) {
```

Comment:
what problem are you trying to solve by adding these? Looks like if `actualValue` is a vartype, it'll pass anyway

Reply:
At this point the vartype is bound to an actual type. I'm not sure if we need to add VarType within shouldAnnotate, but it seems safer to do so.

Reply:
sounds good to me